float64 alpha1
float64 alpha2
bool second_task
